src/modules/popUp.js

```diff
@@ -0,0 +1,56 @@
+const popSection = document.getElementById('popSection');
+const popUp = async (index) => {
+  const data = await fetch('https://www.themealdb.com/api/json/v1/1/filter.php?c=Beef');
+  const json = await data.json();
+  const { meals } = json;
+  const mealData = await fetch(`https://www.themealdb.com/api/json/v1/1/lookup.php?i=${meals[index].idMeal}`);
+  const jsonMeal = await mealData.json();
+  const specificMeal = jsonMeal.meals[0];
+  const section = document.createElement('section');
+  section.className = 'popUp';
+  popSection.appendChild(section);
+  const div = document.createElement('div');
+  div.className = 'popUpDiv';
+  section.appendChild(div);
+  const button = document.createElement('button');
+  button.id = 'buttonX';
+  button.className = 'buttonX';
+  button.innerHTML = '<i class="fa-solid fa-xmark" ></i>';
+  div.appendChild(button);
+  const imgs = document.createElement('img');
+  imgs.src = meals[index].strMealThumb;
+  div.appendChild(imgs);
+  const h3 = document.createElement('h3');
+  h3.innerHTML = meals[index].strMeal;
+  div.appendChild(h3);
+  const p1 = document.createElement('p');
+  p1.innerHTML = `<strong>Area:</strong> ${specificMeal.strArea}`;
+  div.appendChild(p1);
+  const p2 = document.createElement('p');
+  p2.innerHTML = `<strong>Category:</strong> ${specificMeal.strCategory}`;
+  div.appendChild(p2);
+  const p3 = document.createElement('p');
+  p3.innerHTML = `<strong>Instructions:</strong> ${specificMeal.strInstructions}`;
+  div.appendChild(p3);
+  const div1 = document.createElement('div');
+  div1.className = 'commentsContainer';
+  div.appendChild(div1);
+  const h4 = document.createElement('h4');
+  h4.innerHTML = 'Comments';
+  div1.appendChild(h4);
+  const divComments = document.createElement('div');
+  divComments.id = 'divComments';
+  divComments.className = 'divComments';
+  div1.appendChild(divComments);
+  const h4Add = document.createElement('h4');
+  h4Add.innerHTML = 'Add a comment';
+  div.appendChild(h4Add);
+  const form = document.createElement('form');
+  form.innerHTML = '<input type="text" name="name" id="nameF"><textarea name="comment" id="textComment" cols="30" rows="5"></textarea><button id="submit" type="submit">Comment</button>';
+  div.appendChild(form);
+  button.addEventListener('click', () => {
+    popSection.innerHTML = '';
+  });
+};
+
+export default popUp;
```

Comment:
When the Popup shows, it needs to be scrolled down to be viewed. For a better user experience, I'll suggest you make it show entirely on-screen just as in the wireframe. Good job once more with the use of javascript and CSS 👏 

Your design
![ppp](https://user-images.githubusercontent.com/50990925/196555679-24fced76-d029-45ea-adaf-0e4bd4339d1c.png)

Wireframe:
![wireframe](https://user-images.githubusercontent.com/50990925/196556396-72fb457f-b2f0-44c9-a5ea-dfd9e7c85837.png)

Reply:
@Shedrack-Sunday thank you for the review 🙌. I tried to not use the scroll down, but when I did, it gave me a lot of problems related with user experience. So I thought that it was better implement the scroll. Also, I made it because in the future we are going to have a lot of comments there.

Reply:
Alright, I understand. You you maybe signify and arrow to show there's more content? Just add a `div`, so users will be aware there's more bellow. That will be all😊

Reply:
Okay @Shedrack-Sunday this is a good idea. Thank you 👌

Reply:
You're welcome 👏🏿